app/assets/stylesheets/application.scss

```diff
@@ -173,3 +173,33 @@ table {
     line-height: 1.25;
   }
 }
+
+.banner-dangerous {
+  @include bold-19;
+  background: $white;
+  color: $error-colour;
+  border: 5px solid $error-colour;
```

Comment:
then if that pattern updates in govuk elements then we dont have to do anything and its less duplication.